.github/workflows/ci.yml

```diff
@@ -81,7 +81,7 @@ jobs:
   notify-slack:
     runs-on: ubuntu-latest
     needs: [test]
-    if: always() && github.repository == 'linode/linodego' # Run even if integration tests fail and only on main repository
+    if: always() && github.ref == 'refs/heads/main' && github.event_name == 'push' # Run even if integration tests fail and only on main repository
```

Comment:
Do we want to add the condition `github.ref == 'refs/heads/main' && github.event_name == 'push'` along with `github.repository == 'linode/linodego'` to limit slack notifications?

Reply:
I think it's already in `main`? Maybe we can do some main -> proj -> this-branch merges?